### PR TITLE
Add aws lambda event support for image handler

### PR DESCRIPTION
**Pull request by yubozhao**
What changes were proposed in this pull request?
------------------------------------------------
Add aws lambda support for image handler.

* support direct file POST to lambda endpoints.
* support image format jpeg/jpg/png

Does this close any currently open issues?
------------------------------------------

How was this patch tested?
--------------------------
Localhost

### CODE REVIEW

File: bentoml/handlers/image_handler.py [hunk 1]
```diff
@@ -100,4 +104,19 @@ def handle_cli(self, args, func):
...
+            import cv2
+        except ImportError:
+            raise ImportError("opencv-python package is required to use ImageHandler")
+
+        if event['headers']['Content-Type'] in ACCEPTED_CONTENT_TYPES:
+            nparr = np.fromstring(base64.b64decode(event['body']), np.uint8)
+            image = cv2.imdecode(nparr, cv2.IMREAD_COLOR)
+        else:
+            raise BentoMLException(
+                "BentoML currently doesn't support {content_type} for AWS Lambda".format(
```
Comment (parano) [inline bentoml/handlers/image_handler.py]: nit 'BentoML ImageHandler ...'

File: bentoml/handlers/image_handler.py [hunk 2]
```diff
@@ -100,4 +104,19 @@ def handle_cli(self, args, func):
         print(result)
 
     def handle_aws_lambda_event(self, event, func):
-        raise NotImplementedError
+        try:
+            import cv2
+        except ImportError:
+            raise ImportError("opencv-python package is required to use ImageHandler")
+
+        if event['headers']['Content-Type'] in ACCEPTED_CONTENT_TYPES:
```
Comment (parano) [inline bentoml/handlers/image_handler.py]: when Content-Type header is not set by client request, the AttributionError is not handled. maybe use `if event['headers'].get('Content-Type', None) in ACCEPTED_CONTENT_TYPES:` here instead?

**Comment by parano** [comment]
two minor comments, otherwise LGTM

File: bentoml/handlers/image_handler.py (hunk 2 above)
Comment (yubozhao) [reply bentoml/handlers/image_handler.py]: Good suggestion

**Comment by parano** [comment]
👏

**Pull request merged**